app/components/settings/index.js

```diff
@@ -89,47 +89,14 @@ class Settings extends Component {
               <Text style={styles.label}>
                 {I18n.t('settings.yourAreas')}
               </Text>
-
-              {areas.map((item, key) => {
-                const area = item.attributes;
-                area.id = item.id;
-                const image = areasImages[area.id];
-
-                return (
-
-                  <TouchableHighlight
-                    key={key}
-                    activeOpacity={0.5}
-                    underlayColor="transparent"
-                  >
-                    <View style={styles.item}>
-                      <View style={styles.imageContainer}>
-                        {image
-                          ? <Image style={styles.image} source={{ uri: image }} />
-                          : null
-                        }
-                      </View>
-                      <Text style={styles.title} numberOfLines={2}>
-                        {area.name}
-                      </Text>
-                      <TouchableHighlight
-                        activeOpacity={0.5}
-                        underlayColor="transparent"
-                      >
-                        <Image style={Theme.icon} source={nextIcon} />
-                      </TouchableHighlight>
-                    </View>
-                  </TouchableHighlight>
-                );
-              })
-              }
+              <AreaList areas={areas} areasImages={areasImages} navigate={navigate} syncing={syncing} />
```

Comment:
Lets make a container for this new `AreaList` component so we don't have to pass this data to it.